src/main/java/edu/ucsb/cs156/happiercows/helpers/CommonStatsCSVHelper.java

```diff
@@ -0,0 +1,66 @@
+package edu.ucsb.cs156.happiercows.helpers;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+import edu.ucsb.cs156.happiercows.entities.CommonStats;
+
+/*
+ * This code is based on 
+ * <a href="https://bezkoder.com/spring-boot-download-csv-file/">https://bezkoder.com/spring-boot-download-csv-file/</a>
+ * and provides a way to serve up a CSV file containing information associated
+ * with an instructor report.
+ */
+
+public class CommonStatsCSVHelper {
+
+  private CommonStatsCSVHelper() {}
+
+  /**
+   * This method is a hack to avoid a jacoco issue; it isn't possible to 
```

Comment:
This comment is copied from one that I wrote, but I think both the one that I wrote, and the one here have an error in them; I think the issue is not with jacoco, but with pitest.  I think perhaps we should change the comment in both to say pitest instead of jacoco.

Sorry for the inconvenience, but the best time to fix an error is always as soon as you find it (especially if it is starting to proliferate.)  True, it isn't fair that my mistake now has to be fixed by you, but you'll probably find that team software dev is full of little unfairnesses like this.

Reply:
No worries! Will do, Ive just been trying to copy the comments when i see them to keep the code clearer